packages/legend-application-query/src/__lib__/LegendQueryUserDataHelper.ts

```diff
@@ -20,18 +20,20 @@ import { createSimpleSchema, deserialize, list, primitive } from 'serializr';
 
 export enum LEGEND_QUERY_USER_DATA_KEY {
   RECENTLY_VIEWED_QUERIES = 'query-editor.recent-queries',
+  LAST_QUERY_DATASPACE_KEY = 'last_query_dataspace',
```

Comment:
Could we remove `_KEY` from this enum value? Since it's already in the enum name, it seems unnecessary to have it in the enum value, too.